external/CMakeLists.txt

```diff
@@ -34,6 +34,25 @@ target_include_directories(css-color-parser-cpp
   PUBLIC
   ${CMAKE_CURRENT_SOURCE_DIR}/css-color-parser-cpp)
 
+## zlib ##
+if (CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
```

Comment:
I have no intention to support for Clang/GCC under win32 at this time. BTW, do small things in a single step is a better option. Or the reviewed patches are too big. And MSVC obviously is the de-facto compiler under Win32. Support for is is a first priority. If we want to support for CLion or Cygwin, We can do it in other patches.